### DISCUSSION TITLE
Release 1.5.0

### DISCUSSION THREAD

**Discussion by asmunds** [Volue-Public/energy-mesh-python#411] [General]
Hi, we are looking forward to seeing the official release of version 1.5, which has the prebuilt winkerberos wheel. When can we expect this release?

**Comment by tnoczyns-volue** [comment]
Hi, we are planning to release version 1.5 this week.

**Comment by asmunds** [comment]
Awesome, thanks!

**Comment by tnoczyns-volue** [comment]
Hi @asmunds, version 1.5 has just been released:
https://github.com/Volue-Public/energy-mesh-python/releases/tag/v1.5.0

**Comment by asmunds** [comment]
Thanks, I appreciate it!
